src/data/queries/tx.ts

```diff
@@ -50,6 +51,24 @@ export const useTxInfo = ({ txhash, queryKeys, chainID }: LatestTx) => {
   )
 }
 
+export const useOsmosisGas = () => {
+  const { networks } = useNetworks()
+
+  return useQuery(
+    [queryKey.tx.osmosisGas],
+    async () => {
+      const { data } = await axios.get(
```

Comment:
I think we should put everything inside a try - catch, that returns a default value if the LCD ends up being down